lua/cmp_luasnip/init.lua

```diff
@@ -85,11 +90,30 @@ function source:complete(params, callback)
 								filetype = ft,
 								snip_id = snip.id,
 								show_condition = snip.show_condition,
+								auto = false
 							},
 						}
 					end
 				end
 			end
+			local auto_table = require('luasnip').get_snippets(ft, {type="autosnippets"})
+			if auto_table then
+				for j, snip in pairs(auto_table) do
+					if not snip.hidden then
+						ft_items[#ft_items+1] = {
+							word = snip.trigger,
+							label = snip.trigger,
+							kind = cmp.lsp.CompletionItemKind.Snippet,
+							data = {
+								filetype = ft,
+								snip_id = snip.id,
+								show_condition = snip.show_condition,
+								auto = true
+							}
+						}
+					end
+				end
+			end
```

Comment:
I think it would be nice to combine this loop with the previous one, less duplication (maybe one outer loop over ft_table and auto_table?)

Reply:
Missed the comment when I resolved it.
Thanks for the suggestion, yes this is much nicer (more tables but nicer code).

Reply:
Oh, nice :D